test/extended/dr/common.go

```diff
@@ -692,6 +860,21 @@ func runPodAndWaitForSuccess(oc *exutil.CLI, pod *applycorev1.PodApplyConfigurat
 	return err
 }
 
+func runPod(oc *exutil.CLI, pod *applycorev1.PodApplyConfiguration) error {
+	err := e2epod.DeletePodWithWaitByName(oc.AdminKubeClient(), *pod.Name, *pod.Namespace)
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	_, err = oc.AdminKubeClient().CoreV1().Pods(*pod.Namespace).Apply(ctx, pod, metav1.ApplyOptions{FieldManager: *pod.Name, Force: true})
```

Comment:
Is the `Force: true` here to overwrite the pod if the previous named pod exists?

Reply:
I think it's not necessary, removed